demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementController.php

```diff
@@ -2509,11 +2509,18 @@ public function importStatementsFromXls(
             throw new DemosException(self::STATEMENT_IMPORT_ENCOUNTERED_ERRORS);
         } catch (Exception $e) {
             $this->logger->error(self::STATEMENT_IMPORT_ENCOUNTERED_ERRORS, ['exception' => $e]);
-            $this->getMessageBag()->add(
-                'error',
-                'statements.import.error.document.unexpected',
-                ['doc' => $fileInfo->getFileName()]
-            );
+            if (DuplicateInternIdException::class) {
```

Comment:
if (DuplicateInternIdException::class === get_class($e)) I guess this is missing, the rest looks good thanks for using this exception